server/metrics/debug.go

```diff
@@ -38,18 +38,18 @@ func (r *debugReporter) Flush() {
 }
 
 func (r *debugReporter) ReportCounter(name string, tags map[string]string, value int64) {
```

Comment:
Any reason to comment these log lines out?

Reply:
Same as above.